prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    // Your code here
+document.addEventListener('DOMContentLoaded', load);
+
+function load() {
+  return ('load');
+}
+console.log('content loaded!');
+
+window.addEventListener('mousemove', mouse);
+
+function mouse() {
+  return ('mouse');
+}
+console.log('mouse is moving');
+
+var $helloworld = document.querySelector('button.hello-world-btn');
+$helloworld.addEventListener('click', whenClick);
+
+function whenClick() {
+  return ('hello world');
+}
+console.log('hello world');
```

Comment:
Instead of `return` in your functions, move the `console.log` into the functions so that every time the event happens, the `console.log` happens.